docs/api-references/user.md

```diff
@@ -0,0 +1,125 @@
+## User API
+
+### `GET` User
+```
+/api/users/me
+```
+Get information about a current user login.
+
+#### Request header
+| Key | Value |
+|---|---|
+| Accept | application/json |
+| Authorization | {token_type} {access_token} |
+
+#### Sample Response
+```json
+{
+  "data": {
+    "id": 74,
+    "full_name": "dungvan2512",
+    "email": "dungvan@test.email.com",
+    "birthday": null,
+    "gender": 0,
+    "address": null,
+    "phone_number": null,
+    "image": "http://foodmarket.com/images/users/default.jpg",
+    "is_admin": 1,
+    "is_active": 0
+  },
+  "success": true
+}
+```
+
+### `Post` user
+```
+/api/users
+```
+Registry a user
+
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| email | Email | required | email to login |
+| password | Password | required, least at 6 character | password |
```

Comment:
**Required** is not validation rule. Just define **Required** or **Optional**
If you want to clear this key, add more info on Description column.